go/enclave/storage/init/sqlite/001_init.sql

```diff
@@ -67,14 +66,10 @@ create table if not exists batch
     is_canonical   boolean    NOT NULL,
     header         blob       NOT NULL,
     l1_proof_hash  binary(32) NOT NULL,
-    l1_proof       INTEGER, -- normally this would be a FK, but there is a weird edge case where an L2 node might not have the block used to create this batch
```

Comment:
are we saying that this edge case doesn't exist anymore?

Reply:
I think there was logic that depended on l1_proof, but now it works with the l1_proof_hash, and is resilient to whether the block is not there